api/openapi-spec/v1.0.yaml

```
@@ -1268,6 +1302,48 @@ paths:
           description: Success
         default:
           $ref: '#/components/responses/error'
+  '/education/schools/{school-id}/users':
+    get:
+      tags:
+        - educationSchool
+      summary: Retrieves the educationUser resources assigned to an educationSchool.
```

taken from https://learn.microsoft.com/en-us/graph/api/educationschool-list-users?view=graph-rest-1.0&tabs=http
```suggestion
      summary: Get the educationUser resources associated with an educationSchool.
```